src/localize/languages/es.json

```diff
@@ -0,0 +1,76 @@
+{
+  "scheduler": "Programación",
+  "actions": {
+    "add": "añadir dispositivo",
+    "cancel": "cancelar",
+    "next": "siguiente",
+    "save": "guardar",
+    "delete": "borrar"
+  },
+  "instructions": {
+    "no_group_selected": "Selecciona un grupo",
+    "no_entity_selected": "Selecciona una entidad",
+    "no_groups_defined": "No hay grupos definidos",
+    "no_entities_for_group": "No hay entidades en este grupo",
+    "no_actions_for_entity": "No hay acciones para esta entidad",
+    "no_entries_defined": "No hay dispositivos definidos"
+  },
+  "fields": {
+    "group": "Grupo",
+    "entity": "Entidad",
+    "action": "Acción",
+    "days": "Dias",
```

Comment:
Debe ser "Días" // falta la tilde.

______________________________________________

Should be "Días" // with tilde

Reply:
You can use the 
![image](https://user-images.githubusercontent.com/10727862/92976231-f263d500-f43e-11ea-8a04-02ed21635636.png)
button to make changes easily commitable, with ` ```suggestion `.

Reply:
thanks so much.